### PR TITLE
Special job message vol.2

### PR DESCRIPTION
**Pull request by ghost**
Ports: https://github.com/OracleStation/OracleStation/pull/544

Examples of a special notice (EDIT: Different words in the latest version):
image

:cl: ike709 BeloneX
add: Some roles now have reminders of what not to do.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/brigphys.dm
Comment (PowerfulBacon) [inline code/modules/jobs/job_types/brigphys.dm]:
```suggestion
	special_notice = "You are a doctor for Security, not a combat medic. Do not do Security's job."
```

**Review by PowerfulBacon — COMMENTED** [review]
Ngl, I still don't particularly fully like it, but I guess some roles do need it. I'll let another maintainer have a look at it.

File: code/modules/jobs/job_types/brigphys.dm
Comment (PowerfulBacon) [reply code/modules/jobs/job_types/brigphys.dm]: actually Grammarly says both are correct

**Comment by Effemu** [comment]
belone made good pr 😳 
add something like: "Your job is to heal prisoners." to it, so they dont assume its just a mindshielded greytide role

File: code/modules/jobs/job_types/brigphys.dm
Comment (utilbird) [inline code/modules/jobs/job_types/brigphys.dm]: and prisoners

**Review by utilbird — COMMENTED** [review]
Messages are really blunt and bland. Could give them a bit of flavour maybe

**Comment by Sinestia** [comment]
I don't like how hostile and strict it is.

 Like the the brig phys text in the original PR was

> "You are not a security officer, do not do their job for them. However, you can help them if they need immediate assistance."

which is much better. 
I'd probably personally write it as

> "Your primary job is to heal officers and prisoners. You are not to do security's job barring extraordinary circumstances."

**Comment by Effemu** [comment]
> 
> 
> I don't like how hostile and strict it is.
> 
> Like the the brig phys text in the original PR was
> 
> > "You are not a security officer, do not do their job for them. However, you can help them if they need immediate assistance."
> 
> which is much better.
> I'd probably personally write it as
> 
> > "Your primary job is to heal officers and prisoners. You are not to do security's job barring extraordinary circumstances."

average security player doesn't know what "extraordinary circumstances" mean

it has to be blunt, big and red for the monkeybrains playing brig physician to get it

File: code/modules/jobs/job_types/captain.dm
Comment (Archanial) [inline code/modules/jobs/job_types/captain.dm]: This is really weird notice for a captain.

File: code/modules/jobs/job_types/clown.dm
Comment (Archanial) [inline code/modules/jobs/job_types/clown.dm]: OOC

File: code/modules/jobs/job_types/mime.dm
Comment (Archanial) [inline code/modules/jobs/job_types/mime.dm:12]: again too OOC

File: code/modules/jobs/job_types/captain.dm
Comment (ghost) [reply code/modules/jobs/job_types/captain.dm]: > This is really weird notice for a captain.

Not really, a lot of Captains like to play as a validhunting Judge Jury and Executioner.

File: code/modules/jobs/job_types/clown.dm
Comment (ghost) [reply code/modules/jobs/job_types/clown.dm]: > OOC

I don't see how that's bad.

File: code/modules/jobs/job_types/mime.dm
Comment (francinum) [reply code/modules/jobs/job_types/mime.dm:12]: There's no problem with them being OOC. These are pretty much just policy reminders.

File: code/modules/jobs/job_types/clown.dm
Comment (francinum) [reply code/modules/jobs/job_types/clown.dm]: There's no problem with them being OOC. These are pretty much just policy reminders.

**Review by francinum — DISMISSED** [review]

File: code/modules/jobs/job_types/clown.dm
Comment (utilbird) [reply code/modules/jobs/job_types/clown.dm]: Ehh, it still feels pretty hostile. I don't think there would be anything wrong with making it more friendly and IC

**Comment by TheFakeElon** [comment]
Authoritarian station 13

File: code/modules/jobs/job_types/captain.dm
Comment (WhyIsCaeciliusTaken) [inline code/modules/jobs/job_types/captain.dm]: I'd replace this with something more inline with our policies on Captains: "You are not a frontline security officer. Let the frontline officers handle the arrests unless they need your help or they are massively understaffed." as we do allow captains to perform arrests if needed, but not ignore their job to just be sec officer++.

That said, the original message is perfect for the HoP.

File: code/modules/jobs/job_types/detective.dm
Comment (WhyIsCaeciliusTaken) [inline code/modules/jobs/job_types/detective.dm:17]: Same as the captain comment, also add a similar message for the warden as they shouldn't be out of the brig hunting suspects either.

**Review by WhyIsCaeciliusTaken — CHANGES_REQUESTED** [review]

**Comment by francinum** [comment]
These should honestly be adding job support to `policy.txt`, which is currently only used for polystaff stuff.

example code from polystaff https://github.com/BeeStation/BeeStation-Hornet/blob/b3d1ef930be506289c2c0f345557167c834bf7b2/code/modules/projectiles/projectile/magic.dm#L281-L283

File: code/modules/jobs/job_types/captain.dm
Comment (ghost) [reply code/modules/jobs/job_types/captain.dm]: Updated it.

**Comment by ghost** [comment]
This didn't act on Francs review by the way, no idea why the commit suggests that it does, I changed the message the Captain gets instead.

**Comment by ghost** [comment]
> These should honestly be adding job support to `policy.txt`, which is currently only used for polystaff stuff.
> 
> example code from polystaff
> 
> https://github.com/BeeStation/BeeStation-Hornet/blob/b3d1ef930be506289c2c0f345557167c834bf7b2/code/modules/projectiles/projectile/magic.dm#L281-L283

Policy.txt doesn't exist.

**Review by utilbird — CHANGES_REQUESTED** [review]
Some of these are pretty aggressive, particularly the clown's.
OOC isn't a huge problem but it's probably worth making it a bit more flavour-y,

This should be config too, as it's tied into server rules

**Comment by ghost** [comment]
Fun fact: most if not all of these messages are originally by Ike, but fine I'll change them.

**Comment by utilbird** [comment]
if @ike709's fine with this in it's current state then you can disregard my review. This should at the very least be a config option though

**Comment by ghost** [comment]
> if @ike709's fine with this in it's current state then you can disregard my review. This should at the very least be a config option though

Yeah I'll do that later.

**Comment by utilbird** [comment]
Closing due to stale state, feel free to re-open it once it's being worked on again :D

**Pull request closed without merging**
